lib/comment/widgets/comment_list_entry.dart

```diff
@@ -0,0 +1,102 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:lemmy_api_client/v3.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:swipeable_page_route/swipeable_page_route.dart';
+import 'package:thunder/account/bloc/account_bloc.dart';
+import 'package:thunder/core/auth/bloc/auth_bloc.dart';
+import 'package:thunder/core/enums/local_settings.dart';
+import 'package:thunder/core/singletons/preferences.dart';
+import 'package:thunder/post/bloc/post_bloc.dart' as post_bloc;
+import 'package:thunder/post/pages/create_comment_page.dart';
+import 'package:thunder/shared/comment_reference.dart';
+import 'package:thunder/shared/snackbar.dart';
+import 'package:thunder/thunder/bloc/thunder_bloc.dart';
+
+Widget buildCommentEntry(

```

Comment:
Would it be possible to convert this to a Stateless/Stateful Widget? Using Stateless/Stateful widgets are usually more performant than using a function that returns a widget! This applies to the comment, community, user, etc.

https://www.youtube.com/watch?v=IOyq-eTRhvo

Reply:
> Using Stateless/Stateful widgets are usually more performant than using a function that returns a widget!

Good to know! Done!

48313fb964e11194850865ec775de6fdddf2262d

> https://www.youtube.com/watch?v=IOyq-eTRhvo

Very informative, thank you!